mod-password-validator/src/test/resources/domain/passwordvalidator/features/validate.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Test password validate
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapiToken)', 'Accept': 'application/json'  }
+
+  @Undefined
+  Scenario: POST validate should return 200 on success
+    * print 'undefined'
+
+  @Undefined
+  Scenario: POST password should return 422 if validation error
+    * print 'undefined'
+
+  @Undefined
+  Scenario: POST password should return 400 if bad request
+    * print 'undefined'
+
+  @Undefined
+  Scenario: POST password should return 500 if internal server error
```

Comment:
I am interested in how you will simulate 500 error

Reply:
Didn't think about it, but I took this example from other modules